README.md

```diff
@@ -93,8 +102,33 @@ python setup.py install
 MACOSX_DEPLOYMENT_TARGET=10.9 CC=clang CXX=clang++ python setup.py install
 ```
 
-If while building from within an anaconda environment you come across errors similar to the following:
+Alternatively, the build process can build SoX (and codecs such as libmad, lame and flac) statically and torchaudio can link them, by defining `BUILD_SOX` environment variable.
+The build process will fetch and build SoX, liblame, libmad, flac before building extension.
+
+```bash
+# Linux
+BUILD_SOX= python setup.py install
 
+# OSX
+BUILD_SOX= MACOSX_DEPLOYMENT_TARGET=10.9 CC=clang CXX=clang++ python setup.py install
+```
+
+This is known to work on a few major Linux distributions such as Ubuntu and CentOS 7 and macOS.
+If you try this on a new system and found a solution to make it work, feel free to share it by opening and issue.
+
+#### Troubleshooting:
```

Comment:
We can put this section into Wiki too.

Reply:
The installation section should stay in the readme. I do like the folds you are adding, which avoids cluttering part of the wiki, at least when rendered such as on github. I'd say to leave troubleshooting here, with the installation steps, but I'm ok either for this.

Since we may leave some pull requests opened for troubleshooting, the troubleshooting section may be redundant, and more work to keep up-to-date. That would be a vote for just using issues for troubleshooting.